### PR TITLE
Github social icon added

### PR DESCRIPTION
**Pull request by ruchikamuddinagiri**
Issue: #22 
Description: Added the "Login with Github" social icon.

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -169,6 +169,10 @@ $(document).ready(function () {
           classes: "weibo",
           text: "Login with Weibo",
         },
+        {
+          classes: "github",
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:173]: Place this button in the correct alphabetical order.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi there! It all looks good, I just have one remark you can see it in the review below.

File: assets/js/buttons-examples.js (hunk 1 above)
Comment (ruchikamuddinagiri) [reply assets/js/buttons-examples.js]: Done! have committed the changes.

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @ruchikamuddinagiri for code

**Pull request merged**